src/safariservices.cs

```diff
@@ -456,4 +467,37 @@ interface SFUniversalLink {
 		bool Enabled { [Bind ("isEnabled")] get; set; }
 	}
 #endif
+
+	[Static]
+	[iOS (15,0), Mac (11,0), MacCatalyst (15,0), NoTV, NoWatch]
+	[DisableDefaultCtor]
+	interface SFExtension {
+		[Field ("SFExtensionMessageKey")]
+		NSString MessageKey { get; }
+	}
+
+	[iOS (15,0), MacCatalyst (15,0), NoMac, NoTV, NoWatch]
+	[BaseType (typeof (NSObject))]
+	[DisableDefaultCtor]
+	interface SFSafariViewControllerActivityButton : NSCopying, NSSecureCoding
+	{
+		[Export ("initWithTemplateImage:extensionIdentifier:")]
+		[DesignatedInitializer]
+		IntPtr Constructor (UIImage templateImage, string extensionIdentifier);
+
+		[NullAllowed, Export ("templateImage", ArgumentSemantic.Copy)]
+		UIImage TemplateImage { get; }
+
+		[NullAllowed, Export ("extensionIdentifier")]
+		string ExtensionIdentifier { get; }
+	}
+
+	[iOS (15,0), MacCatalyst (15,0), NoMac, NoTV, NoWatch]
+	[BaseType (typeof (NSObject))]
+	[DisableDefaultCtor]
+	interface SFSafariViewControllerPrewarmingToken : NSCoding, NSSecureCoding
```

Comment:
Note to TJ,
These protocols may need to be removed from here and this interface may need to be documented in iOSApiProtocolTest.cs